packages/utilities/src/object.test.ts

```diff
@@ -1,4 +1,55 @@
-import { assign, filteredAssign, mapEnumByName, values, omit } from './object';
+import { assign, filteredAssign, mapEnumByName, values, omit, shallowCompare } from './object';
+
+describe('shallowCompare', () => {
+  it('returns true for matching objects', () => {
+    const a = {
+      a: 1,
+      b: 'string',
+      c: {
+        d: 2,
+      },
+    };
+
+    const b = { ...a };
```

Comment:
I think there is another variation of this test that is probably worth adding to the suite:

```ts
const a = {
  a: 1,
  b: 'string',
  c: {
    d: 2,
  },
};

const b = { ...a, c: {...a.c} };

expect(shallowCompare(a, b)).toBeFalsey();
```

Even though the objects have the same keys and values they are not equal to `shallowCompare()` because `a.c !== b.c`.